src/KafkaFlow.Retry/Durable/Definitions/Builders/RetryDurableEmbeddedClusterDefinitionBuilder.cs

```diff
@@ -127,19 +127,12 @@ public RetryDurableEmbeddedClusterDefinitionBuilder WithRetryTypedHandlers(Actio
                         .WithPartitionsAssignedHandler(
                             (resolver, partitionsAssignedHandler) =>
                             {
-                                var log = resolver.Resolve<ILogHandler>();
```

Comment:
Just to confirm: is this change expected?

Reply:
Yes, KafkaFlow is logging something on assign and revoking the partitions. It's not needed.

Reply:
👍